social-tipping-app/src/store/index.js

```diff
@@ -82,13 +111,38 @@ export default new Vuex.Store({
           const loginUserAmount = doc.data().amount
 
           commit('setUserData', {email: payload.email, password: payload.password, uid: uid, userName: loginUserName, amount: loginUserAmount})
-
-          //ダッシュボードに遷移
-          router.push('/dashboard')
         })
         .catch((error) => {
           console.log('Error getting document:', error)
         })
+
+        //全てのユーザと残高を取得
+        db.collection('users')
+          .get()
+          .then((querySnapshot) => {
+
+            const allUsers = []
+
+            querySnapshot.forEach((doc) => {
+
+              const userData = {
+                uid: doc.id,
+                userName: doc.data().userName,
+                amount: doc.data().amount
+              }
+
+              allUsers.push(userData)
+              
+            })
```

Comment:
こちらも同様です！
また、少し余計な空行が目立ちます。
仮にこのままの処理だとしても、
```suggestion
            const allUsers = []
            querySnapshot.forEach((doc) => {
              const userData = {
                uid: doc.id,
                userName: doc.data().userName,
                amount: doc.data().amount
              }
              allUsers.push(userData)
            })
```
のように空行を削除すると可読性が高まります。

基本的には空行は全て無しでも良いのですが、一つのまとまりを表す時に入れる時があります。
今回の処理ですと、「値をループさせて配列に格納する」という意味で一まとまりですので、その観点からも空行が無い方が可読性が高いです。